### ISSUE TITLE
GemStone builds fail if deployments are enabled

### ISSUE THREAD

**Issue by fniephaus** [hpi-swa/smalltalkCI#186]

**Comment by dalehenrich** [comment]
I'm not sure that I am interested in deploying GemStone this way ... at this point in time, I prefer people to follow the GsDevKit_home procedure for installing as a pre-buiilt structure, but leaving this as an open issue is fine with me:)

**Comment by fniephaus** [comment]
I just quickly opened this issue as a reminder and let me rephrase it: If deployments are enabled, GemStone builds currently fail, because the image cannot be found. We should fix this, not add support for GemStone images ;)

**Comment by fniephaus** [comment]
Closing this in favor of #475.
